oracle/README.md

```diff
@@ -52,20 +48,14 @@ When you run `datadog-agent info` you should see something like the following:
 
 ### Compatibility
 
-The oracle check is currently compatible with the linux and darwin-based OS
+The Oracle check is currently compatible with Linux and macOS.
```

Comment:
Capitalize the M in macOS.

Reply:
Nevermind. I see that macOS is how Apple does it.

Reply:
Can confirm that "macOS" is the new official spelling.